pkg/cloud/mock/mock.go

```diff
@@ -457,6 +457,26 @@ func UpdateRegionBackendServiceHook(ctx context.Context, key *meta.Key, obj *ga.
 	return nil
 }
 
+// UpdateRegionBackendServiceHook defines the hook for updating a Region
+// BackendsService. It replaces the object with the same key in the mock with
+// the updated object.
+func UpdateAlphaRegionBackendServiceHook(ctx context.Context, key *meta.Key, obj *ga.BackendService, m *cloud.MockAlphaRegionBackendServices) error {
+	_, err := m.Get(ctx, key)
+	if err != nil {
```

Comment:
Through the error unless it is actually a NotFound errors

Reply:
Okay I updated the error handling, not sure if that's what you had in mind

Reply:
you need to fix this as well?

Reply:
Good catch.  Since i'm making this change anyway I can also just update the other errors.

Reply:
Done